src/common/merge.ts

```diff
@@ -99,3 +103,13 @@ export const mergeWithRetry = async (
     logDebug(`Original error: ${(error as Error).toString()}.`);
   }
 };
+
+export const shouldMerge = (prTitle: string): boolean => {
+  const mergePreset = parseInputMergePreset();
+
+  if (mergePreset === undefined) {
+    return true;
+  }
+
+  return checkPullRequestTitleForMergePreset(prTitle, mergePreset);
+};
```

Comment:
This was previously only used in `pullRequestHandle`. I've refactored it out to a common file so that it can be used by other handlers.